graphrag_sdk/test_metrics.py

```
@@ -0,0 +1,545 @@
+        self.verbose_mode = verbose_mode
+        
+    def measure(
+        self,
+        test_case: LLMTestCase,
+        _show_indicator: bool = True,
+    ) -> float:
+        check_llm_test_case_params(test_case, required_params, self)
+
+        self.evaluation_cost = 0 if self.using_native_model else None
+        with metric_progress_indicator(self, _show_indicator=_show_indicator):
+            self.verdicts: List[ContextualRecallVerdict] = (
```

Ruff (0.8.2)

`ContextualRecallVerdict` may be undefined, or defined from star imports

(F405)